jupyter_notebooks/Analyzing public and hybrid cloud networks .ipynb

```diff
@@ -0,0 +1,567 @@
+      ],
+      "text/plain": [
+       "Trace(disposition='ACCEPTED', hops=[Hop(node='internet', steps=[Step(detail=OriginateStepDetail(originatingVrf='default'), action='ORIGINATED'), Step(detail=RoutingStepDetail(routes=[{'network': '13.59.144.125/32', 'nextHopIp': '240.1.1.5', 'protocol': 'bgp'}], arpIp='240.1.1.5', outputInterface='~Interface_5~'), action='FORWARDED'), Step(detail=ExitOutputIfaceStepDetail(outputInterface='~Interface_5~', transformedFlow=None), action='TRANSMITTED')]), Hop(node='aws-backbone', steps=[Step(detail=EnterInputIfaceStepDetail(inputInterface='~Interface_6~', inputVrf='default'), action='RECEIVED'), Step(detail=RoutingStepDetail(routes=[{'network': '13.59.144.125/32', 'nextHopIp': '240.0.0.0', 'protocol': 'bgp'}], arpIp='240.0.0.0', outputInterface='~Interface_1~'), action='FORWARDED'), Step(detail=ExitOutputIfaceStepDetail(outputInterface='~Interface_1~', transformedFlow=None), action='TRANSMITTED')]), Hop(node='igw-02fd68f94367a67c7', steps=[Step(detail=EnterInputIfaceStepDetail(inputInterface='backbone', inputVrf='default'), action='RECEIVED'), Step(detail=TransformationStepDetail(transformationType='DEST_NAT', flowDiffs=[FlowDiff(fieldName='dstIp', oldValue='13.59.144.125', newValue='10.20.1.207')]), action='TRANSFORMED'), Step(detail=RoutingStepDetail(routes=[{'network': '10.20.1.0/24', 'nextHopIp': '169.254.0.1', 'protocol': 'static'}], arpIp='169.254.0.1', outputInterface='subnet-06a692ed4ef84368d'), action='FORWARDED'), Step(detail=ExitOutputIfaceStepDetail(outputInterface='subnet-06a692ed4ef84368d', transformedFlow={'dscp': 0, 'dstIp': '10.20.1.207', 'dstPort': 80, 'ecn': 0, 'fragmentOffset': 0, 'ingressNode': 'internet', 'ingressVrf': 'default', 'ipProtocol': 'TCP', 'packetLength': 0, 'srcIp': '240.1.1.2', 'srcPort': 49152, 'state': 'NEW', 'tag': 'tag', 'tcpFlagsAck': 0, 'tcpFlagsCwr': 0, 'tcpFlagsEce': 0, 'tcpFlagsFin': 0, 'tcpFlagsPsh': 0, 'tcpFlagsRst': 0, 'tcpFlagsSyn': 0, 'tcpFlagsUrg': 0}), action='TRANSMITTED')]), Hop(node='subnet-06a692ed4ef84368d', steps=[Step(detail=EnterInputIfaceStepDetail(inputInterface='igw-02fd68f94367a67c7', inputVrf='default'), action='RECEIVED'), Step(detail=RoutingStepDetail(routes=[{'network': '10.20.1.0/24', 'n
```

Comment:
Replace starting sentence with - "In both this and the other reachability query..."